### PR TITLE
Add SendOwl

### PR DESCRIPTION
**Pull request by Poukame**

### CODE REVIEW

File: img/s/sendowl.com.svg [hunk 1]
```diff
@@ -0,0 +1 @@
....8c-.1-.1-.3-.2-.5-.2zm4.9 5.1c-.3 0-.6.3-.5.6v.6c1.2 11 10.4 19.5 21.6 19.7h7.4c.2 0 .4-.1.4-.3V54.6c0-.2-.1-.4-.3-.4H41.5c-9.5-.2-17.3-7-19-16q-.1-.3-.2-1.1 0-.2-.1-.4h-.1l-1.4-.9q-.1-.1-.4-.1zm5.5 2.7h-.1c-.3.1-.5.4-.4.8q0 .2.1.3c2.1 6.9 8.4 12 16 12.2h7.4c.2 0 .4-.1.4-.3v-1.8c0-.2-.1-.3-.3-.4H41.5c-6-.1-11.1-3.8-13.3-9q-.1-.3-.3-1-.1-.2-.3-.3l-.1-.1-1.5-.4h-.1zm5.7.9q-.1 0-.2.1c-.3.2-.5.5-.3.8q.1.3.2.5c2.1 3.5 5.9 5.9 10.3 6.1H48.8c.2 0 .4-.2.4-.4v-1.7c0-.2-.1-.3-.3-.4h-7.3c-3-.1-5.7-1.5-7.4-3.7q-.3-.3-.7-1-.2-.2-.4-.2l-.1-.1zm6.9-1h-.1l-1.4.4q-.1.1-.3.2c-.2.2-.2.6 0 .8l.6.6c1.2.9 2.6 1.5 4.2 1.5l.4.1h7c.2 0 .3-.2.4-.4V39.8c0-.2-.2-.4-.4-.4h-7.3q-.9-.1-1.7-.4l-.9-.6q-.2-.1-.5-.1zM16.1 13.2l-.2.3v-.2c-.8 2-1.2 4.1-1.2 6.4 0 9.5 7.7 17.2 17.2 17.2 9.6 0 17.3-7.7 17.3-17.2 0-2.1-.4-4.1-1.1-5.9l-.2-.5v.2C44.5 4.4 32.8 8.6 32.8 16c0 1.2-1.7 1.2-1.7 0 0-7.3-11.2-12.2-15-2.8zm32.7 21.3h-3.9l-.1.1-2.1 1.4q-.2.2-.2.4c0 .3.2.5.4.5H48.8c.2 0 .3-.1.4-.3V34.8c-.1-.2-.2-.3-.4-.3zM23.3 19.7c-2 0-3.7-1.6-3.7-3.7 0-2 1.7-3.7 3.7-3.7 2.1 0 3.7 1.7 3.7 3.7 0 2.1-1.6 3.7-3.7 3.7zm17.3 0c-2.1 0-3.7-1.6-3.7-3.7 0-2 1.6-3.7 3.7-3.7 2 0 3.6 1.7 3.6 3.7 0 2.1-1.6 3.7-3.6 3.7z" style="fill:#222"/></svg>
```
Comment (Carlgo11) [inline img/s/sendowl.com.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 64 64"><path fill="#222" d="M32 0c10.9 0 19.7 8.8 19.7 19.7v43.4c0 .5-.4.9-.9.9h-9a29.5 29.5 0 0 1-29.5-29.5V19.7C12.3 8.8 21.1 0 32 0zM15.4 30.6c-.3 0-.6.2-.6.5v3.8a27 27 0 0 0 26.6 26.6h7.4c.2 0 .4-.1.4-.3v-1.7c0-.2-.1-.4-.3-.4h-7.5a24.8 24.8 0 0 1-24.2-24.2v-2.3h-.1l-1.2-1.8a.8.8 0 0 0-.5-.2zm4.9 5.1c-.3 0-.6.3-.5.6v.6a22.1 22.1 0 0 0 21.6 19.7h7.4c.2 0 .4-.1.4-.3v-1.7c0-.2-.1-.4-.3-.4h-7.4c-9.5-.2-17.3-7-19-16l-.2-1.1-.1-.4h-.1l-1.4-.9-.4-.1zm5.5 2.7h-.1c-.3.1-.5.4-.4.8l.1.3c2.1 6.9 8.4 12 16 12.2h7.4c.2 0 .4-.1.4-.3v-1.8c0-.2-.1-.3-.3-.4h-7.4c-6-.1-11.1-3.8-13.3-9l-.3-1-.3-.3-.1-.1-1.5-.4h-.1zm5.7.9-.2.1c-.3.2-.5.5-.3.8l.2.5c2.1 3.5 5.9 5.9 10.3 6.1h7.3c.2 0 .4-.2.4-.4v-1.7c0-.2-.1-.3-.3-.4h-7.3c-3-.1-5.7-1.5-7.4-3.7l-.7-1-.4-.2-.1-.1zm6.9-1h-.1l-1.4.4-.3.2c-.2.2-.2.6 0 .8l.6.6a7 7 0 0 0 4.2 1.5l.4.1h7c.2 0 .3-.2.4-.4v-1.7c0-.2-.2-.4-.4-.4h-7.3q-.9-.1-1.7-.4l-.9-.6-.5-.1zM16.1 13.2l-.2.3v-.2a17.2 17.2 0 0 0 16 23.6 17.2 17.2 0 0 0 16.2-23.1l-.2-.5v.2C44.5 4.4 32.8 8.6 32.8 16c0 1.2-1.7 1.2-1.7 0 0-7.3-11.2-12.2-15-2.8zm32.7 21.3h-3.9l-.1.1-2.1 1.4-.2.4c0 .3.2.5.4.5h5.9c.2 0 .3-.1.4-.3v-1.8c-.1-.2-.2-.3-.4-.3zM23.3 19.7a3.7 3.7 0 1 1 0-7.4c2.1 0 3.7 1.7 3.7 3.7 0 2.1-1.6 3.7-3.7 3.7zm17.3 0a3.6 3.6 0 0 1-3.7-3.7c0-2 1.6-3.7 3.7-3.7 2 0 3.6 1.7 3.6 3.7 0 2.1-1.6 3.7-3.6 3.7z"/></svg>
```

File: entries/s/sendowl.com.json [hunk 2]
```diff
@@ -0,0 +1,13 @@
+{
+    "SendOwl": {
+      "domain": "sendowl.com",
+      "url": "https://www.sendowl.com",
```
Comment (Carlgo11) [inline entries/s/sendowl.com.json]:
```suggestion
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: entries/s/sendowl.com.json [hunk 3]
```diff
@@ -0,0 +1,12 @@
+{
+    "SendOwl": {
+      "domain": "sendowl.com",
+      "tfa": [
+        "totp"
+      ],
+      "documentation": "https://help.sendowl.com/help/two-factor-authentication",
+      "keywords": [
+        "other"
```
Comment (Carlgo11) [inline entries/s/sendowl.com.json:9]: Is it similar to Stripe? Because Stripe is in the Payments category.
```suggestion
        "payments"
```

Comment (Poukame) [reply entries/s/sendowl.com.json:9]: No, it's not like Stripe. Stripe is payment processor. SendOwl is about product selling flow.
SendOwl is more like ThriveCart which is in "other" category.

**Review by Carlgo11 — APPROVED** [review]
Thanks for the pull request! :+1:

**Pull request merged**